Test/Framework/Diff.hs

```diff
@@ -187,7 +225,13 @@ Haskell diff, in case the diff tool is not present
 -}
 multiLineDiffHaskell :: String -> String -> ColorString
 multiLineDiffHaskell left right =
-    noColor $ ppDiff $ D.getGroupedDiff (lines left) (lines right) -- this code is now part of the Diff library (hence the >0.3 in Cabal)
+    if length left > maxLen || length right > maxLen
+    then noColor
+             ("Refusing the compute a multiline diff for strings with more than " ++ show maxLen ++
```

Comment:
There is a typo "Refusing the compute" --> "Refusing to compute"

Reply:
Thanks! See #88